app/poros/image.rb

```diff
@@ -0,0 +1,7 @@
+class Image
+  attr_reader :url, :id
+  def initialize(image_data)
+    @url = image_data[:results].first[:urls][:full]
+    @id = nil
```

Comment:
I wasn't sure if it was totally necessary to make this class because it only holds URL, but I decided to make it in case I want to do more with the image in the future.